optimism/op-node/rollup/driver/state.go

```diff
@@ -221,7 +221,7 @@ func (s *Driver) eventLoop() {
 
 	// Create a ticker to check if there is a gap in the engine queue. Whenever
 	// there is, we send requests to sync source to retrieve the missing payloads.
-	syncCheckInterval := time.Duration(s.Config.BlockTime) * time.Second * 2
+	syncCheckInterval := time.Duration(s.Config.BlockTime) * time.Second / 10
```

Comment:
We should add a FIXME here

Reply:
[09c586e](https://github.com/gattaca-com/based-op/pull/119/commits/09c586e5788ff5483664c0f6736d21c0214576b1)